src/Loader/EnumLoader.php

```diff
@@ -44,6 +44,10 @@ public function load($resource, $locale, $domain = 'messages')
         foreach ($consts as $const => $value) {
             $key = $this->getPrettyName($domain) . "." . strtolower($const);
 
+            if (is_array($value)) {
```

Comment:
What if we move the is_array() check to the first line inside the loop? Then we can skip a bit of useless work :smile: